clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -3602,6 +3602,7 @@ void CodeGenFunction::EmitCheck(
   llvm::Value *RecoverableCond = nullptr;
   llvm::Value *TrapCond = nullptr;
   bool NoMerge = false;
+  bool SanitizeGuardChecks = ClSanitizeGuardChecks;
```

Comment:
Usually we parse Cls as `if (ClSanitizeGuardChecks.count())  override the value calculated normal way.`

Reply:
Noted for the next patch (this patch has been updated to be Driver changes only).